src/components/Song/Upvote.js

```diff
@@ -0,0 +1,20 @@
+import { contains } from 'ramda'
+import React from 'react'
+import Arrow from '../../img/arrow-up.svg'
+import { readVotesFromStorage } from '../../util/votes'
```

Comment:
relative imports can get kind of tricky when you get pretty far down the component tree. you should be able to import from `util/votes` since we're using `NODE_PATH=src/`.

Reply:
Good call! I forgot we had that. Just tested it out and it works fine with `util/votes` and its way less error-prone